code/__HELPERS/constants.dm

```diff
@@ -1,2 +1,2 @@
-var/global/const/TICKS_IN_DAY = 864000
-var/global/const/TICKS_IN_SECOND = 10
+var/const/TICKS_IN_DAY = 864000
```

Comment:
There is
literally
no way to instantiate `world` more than twice
`world` is a singleton
`world` holds all the server info
you can only have a new `world` by starting a separate server in which case it's a seperate execution environment so you dont have two worlds you have two servers